### ISSUE TITLE
IBM-combobox-AVT issue

### ISSUE THREAD

**Issue by aishwaryamyana** [IBM/carbon-components-angular#1616]
Accessibility violation issues found un combobox .Please find attached screenshots.
1. Verify the <div> element with "listbox" role has keyboard access
2. The element with WAI-ARIA role of "listbox" does not contain or own at least one child element with each of the following WAI-ARIA roles: "option"
3.The 'autocomplete' attribute's token(s) are not appropriate for the input form field

<img width="1680" alt="Screen Shot 2020-11-18 at 10 46 20 AM" src="https://user-images.githubusercontent.com/53562295/99487075-8e541480-298b-11eb-920e-8e882c22afc4.png">
<img width="1678" alt="Screen Shot 2020-11-18 at 10 46 08 AM" src="https://user-images.githubusercontent.com/53562295/99487089-93b15f00-298b-11eb-9efb-c632dce55941.png">

**Comment by cal-smith** [comment]
Fixed by #1641
